lilit.sargsyan/repo/Lesson_3/exercise_17/exercise.py

```diff
@@ -0,0 +1,21 @@
+num_1 = int(input('Input the first integer:'))
+num_2 = int(input('Input the second integer:'))
+num_3 = int(input('Input the third integer:'))
+sum_of_numbers = num_1 + num_2 + num_3
+print('Sum is', int(sum_of_numbers))
+average_of_numbers = sum_of_numbers / 3
+print('Average is', int(average_of_numbers))
+product_of_numbers = num_1 * num_2 * num_3
+print('Product is', int(product_of_numbers))
```

Comment:
LW: The multiple of the integers numbers are already integer type. No need to convert to integer